server/db/actions/Dog.js

```diff
@@ -0,0 +1,22 @@
+import dbConnect from "../dbConnect";
+import Dog from "../models/Dog";
+
+export async function getDogs() {
+  await dbConnect();
```

Comment:
Move`dbConnect()` into the try/catch block so the technical mongoose error does not propagate to the client